[IMP] payroll: made targeted edits for 16

larm-odoo opened the pull request:
Making edits for v 16 - screens/UI changed, some sections added and names for some things changed.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/5449).

jero-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -59,25 +59,21 @@ are correctly entered.
 ~~~~~~~~~~~~~~~~~~~
 
-To create a new work entry type, click the :guilabel:`Create` button. Enter the information on the
+To create a new work entry type, click the :guilabel:`New` button, and enter the information on the
```
I think Work Entry Type should have a guilabel, or at least be capitalized/italicized when it is first introduced. (It appears to have a guilabel in some of the other Payroll docs, but that is specific to a field, so I realize there is some gray area)

larm-odoo replied:
I honestly don't recall the logic on why we didn't here- I think it was because it was being used as a 'generic' phrase but I totally agree with you .

jero-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -91,32 +87,39 @@ form:
+- Check boxes: If any of the items in the list applies to the work entry, check off the box by
+  clicking it. If :guilabel:`Time Off` is checked off, a :guilabel:`Time Off Type` field appears.
+  This field has a drop-down menu to select the specific type of time off, or a new type of time off
+  can be entered.
```
There is another check box, the "Keep Time Off Right" field. Can you elaborate on it? (The tool tip does not really help much, at least it did not help me). After some back and forth with translating it seems like this might be closer to "entitlement" instead of "right".

larm-odoo replied:
I am checking with Quentin on this- I will have this updated when I hear back.

larm-odoo replied:
OK, this is for Belgium only and affects if time off can be rolled over into the next year, and if it affects how much time they accrue, etc.

jero-odoo reviewed (changes requested):
Let me know if you want to go over any of these, especially the "Leave Right" field. I think it is a translation issue that is making it a bit complicated. Thanks!

larm-odoo commented:
Ready for you, @tiku-odoo!

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -257,10 +267,9 @@ like expenses, reimbursements, or deductions. These other inputs can be configur
-the :guilabel:`Code`, and which structure it applies to in the :guilabel:`Availability in Structure`
-field. Click the :guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete
-the entry.
+To create a new input type, click the :guilabel:`New` button. Enter the :guilabel:`Description`, the
```
Line 264 include sub menu

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -257,10 +267,9 @@ like expenses, reimbursements, or deductions. These other inputs can be configur
-field. Click the :guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete
-the entry.
+To create a new input type, click the :guilabel:`New` button. Enter the :guilabel:`Description`, the
+:guilabel:`Code`, and which structure it applies to in the :guilabel:`Availability in Structure`
+field.
```
What does entering these fields mean? I can see that we need to do this. The documentation should have extra info for the user.

tiku-odoo replied:
As a layman to Payroll this doc is somewhat confusing to me. Pretend you're writing to someone who is trying to configure Odoo on their own.

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -311,15 +314,25 @@ The required fields for an advantage are:
+.. image:: payroll/new-advantage.png
+   :align: center
+   :alt: List of advantages employee's can have.
 
```
It would be nice to have some background on the options we are picking here. These forms can be overwhelming. Remember to add sections.

larm-odoo replied:
I thought the explanation of what an advantage was, and why you'd add one was enough and clear. I did add the sections and included the non-required information to add as well, so that part is taken care of!

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -332,23 +345,24 @@ Personal Info`.
-To edit an entry, select it from the list. Then, click the :guilabel:`Edit` button, and modify the
-entry. When done, click :guilabel:`Save` or :guilabel:`Discard` to save the information or cancel
-the edits.
+To edit an entry, select it from the list, and modify the entry. To make a new entry, click the
```
line 341 add in section

larm-odoo replied:
After making so many changes, I think this section is for the personal info section -  if so, I don't think it needs sections since it calls out the required info, and the information seems understandable and clear to me

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -332,23 +345,24 @@ Personal Info`.
-To edit an entry, select it from the list. Then, click the :guilabel:`Edit` button, and modify the
-entry. When done, click :guilabel:`Save` or :guilabel:`Discard` to save the information or cancel
-the edits.
+To edit an entry, select it from the list, and modify the entry. To make a new entry, click the
```
Instead of "entry" let's specify the info specifically.

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -332,23 +345,24 @@ Personal Info`.
-   :align: center
-   :alt: New personal information entry.
+The required fields, aside from entering the :guilabel:`Information` name, are
+:guilabel:`Related Field` and :guilabel:`Category`. Select a :guilabel:`Related Field` from the
+drop-down menu that best describes what kind of information this entry is, and where it is going to
+be stored in the backed. Select a :guilabel:`Category` from the drop-down menu that the information
+should be under, such as :guilabel:`Address` or :guilabel:`Personal Documents`.
 
 The two most important fields on the personal info form are :guilabel:`Is Required` and
 :guilabel:`Display Type`. Checking the :guilabel:`Is Required` box makes the field mandatory on the
-employee's card.
+employee's card. The :guilabel:`Display Type` drop-down menu allows for the information to be
+entered in a variety of ways, from a :guilabel:`Text` box, to a customizable :guilabel:`Radio`
+button, a :guilabel:`Checkbox`, a :guilabel:`Document`, and more.
```
This paragraph isn't in line with the rest of the sections, the others are listed. I like the detail in this section though!

tiku-odoo replied:
Zac usually prefers explanations like this, so may be good to go off of and model the other sections like this. They lacked depth.

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -357,3 +371,27 @@ Resume
    Currently, the :guilabel:`Resume` feature found inside the :menuselection:`Payroll app -->
```
I wouldn't lead with an admonition black, it needs a summary before.

larm-odoo replied:
This was suggested by Zac since this is not a super functional section for most companies who use Odoo.  I believe this is going to change in the next big release, so this can be edited when this is expanded/changed in the next version. 

I do agree with you- typically, we would not have this, but I put this in since it was specifically told that I could do this as a work-around for this particular section.

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -357,3 +371,27 @@ Resume
    Configuration` menu is still in development and only serves a specific use case for Belgian
    markets. The documentation will be updated when this section has matured to more markets.
+
+Offers
```
I don't see this in runbot 16.3

tiku-odoo replied:
Good to include sections so reader has direction

larm-odoo replied:
Might be because this was in 16.4, not 16.3. I checked and I do see it in 16.4.

StraubCreative replied:
@larm-odoo can you get clarity from support as to whether or not 16.4 is the stable release for 16 or if it is staging for 17? That will answer your question as to whether or not to include here for Odoo 16 doc.

larm-odoo replied:
I asked Thoma (support person for payroll) and he said if you make a new trial database online (potential cusotmers0 they will be served a database in 16.4. If they create a database hosted on odoo SH they will be in version 16.0. So I believe form this info, that means we can use the 16.4 version, since new trials are all done in this.

tiku-odoo commented on content/applications/hr/payroll/advantages.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Alt text updated

tiku-odoo commented on content/applications/hr/payroll/new-advantage.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
I was told by Sam and Zac that I could use the 933 when I needed a wider picture to keep the information crisp and clear (it's the max width). I had a lot of issues with other docs that had lots of text and would become too blurry (even zooming in for the screen grab, when it was resized then compressed they were a mess).  So for this one, with all the text and comparing it to the smaller/less wide ones I tried- these were better so I kept to this size.  I will update the alt text for these, but the size is OK (this size has been published in other docs I have published).

StraubCreative replied:
933 is ok 👍

tiku-odoo commented on content/applications/hr/payroll/new-rule.png:
Image lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Alt text updated

tiku-odoo commented on content/applications/hr/payroll/new-structure.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Alt text updated

tiku-odoo commented on content/applications/hr/payroll/new-work-entry.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Alt text updated

tiku-odoo commented on content/applications/hr/payroll/offers.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

tiku-odoo commented on content/applications/hr/payroll/other-input.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Updated alt text

tiku-odoo commented on content/applications/hr/payroll/personal-info.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Updated alt text

tiku-odoo commented on content/applications/hr/payroll/personal-new.png:
Image is larger than 768 px wide.

tiku-odoo commented on content/applications/hr/payroll/rules.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Updated alt text

tiku-odoo commented on content/applications/hr/payroll/salary-structure.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Updated alt text

tiku-odoo commented on content/applications/hr/payroll/structure-regular-pay-rules.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

tiku-odoo commented on content/applications/hr/payroll/structure-type.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Updated alt text

tiku-odoo commented on content/applications/hr/payroll/work-entry-types.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Updated alt text

tiku-odoo commented on content/applications/hr/payroll/working-schedules.png:
Image is larger than 768 px wide and lacks any annotation that might be helpful for the reader following a process.

larm-odoo replied:
Updated alt text

tiku-odoo reviewed (commented):
@larm-odoo 

Great start to this doc. I think it needs some work still. A lot of what I'm reading is what I see on the screen. I want to know more and understand why?. I would like to have a complete understanding of what impact the settings have. 

Also, there is a heavy reliance on the images. 

Images **should** be under 768 and include annotations to direct the customer. Try reshaping the browser. 

Also can you verify what version this is for- b/c I'm not seeing Offers, line 226 is "add a line", and working schedules in working times.  

Can you give it another rewrite and I'll take another pass at it? 

Thanks and an awesome start to a helpful doc. Can't see what you come up with next. 

Tim

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -91,32 +88,41 @@ form:
-Working Times`. The working times that are available for an employee's contracts and work entries
-are found in this list.
+To view the currently configured working schedules, go to :menuselection:`Payroll --> Configuration
+--> Working Schedules`. The working schedules that are available for an employee's contracts and
+work entries are found in this list.
```
Hi Tim- this is n 16.4, and in that DB, the title changes to working schedules from working times. So if you look at this in 16.4 it should be correct.

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -185,13 +195,13 @@ useful to add may be a `Bonus`.
 To view all the various structures for each structure type, go to :menuselection:`Payroll -->
 Configuration --> Structures`.
```
OK thanks- I was just typing what they clicked on, thinking it wasn't needed to add that sub, or that it would be confusing- but I'll remember this for the future

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -59,25 +59,22 @@ are correctly entered.
+- :guilabel:`External Code`: This code is used for exporting data to a third-party payroll service.
+  Refer to the third-party being used in order to determine the code to enter for the specific work
+  type entry.
+- :guilabel:`SDWorx code`: This code is only for companies that use SDWorx, a payroll service
+  provider.
```
Correct - this would be used only when exporting the information in a spreadsheet, that you would then import to the third-party. It's just so the third-party can read/see the data, then know where to put whatever data in their app, not an actual integration. It's really a work-around = )

larm-odoo commented on content/applications/hr/payroll.rst:
Beneath the section I explained what the different rounding methods mean (up, down, etc), then below that is an example block. I thought that was best- to have it go from broad to granular, in the order it is displayed.  So I think the way it is now is clear, but you just have to read the whole 'chunk'.

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -91,32 +88,41 @@ form:
 
-Working times
--------------
+- :guilabel:`Unpaid in Structures Types`: If the work entry is for unpaid work, specify which pay
+  structure the work entry applies to from the drop-down menu.
```
These would be used for unpaid workers such as volunteers, interns, or people who might be working at the facility/company but being paid by someone else (like a visiting worker?) - not sure about that last one, but it's when you need to do a payslip entry to log their time but they are not being paid for whatever (legal) reason . Could be when you have a job that does unpaid training or something.

larm-odoo replied:
I added that to clarify this- because you are right, it honestly doesn't make sense at first glance.  Odoo is usually for businesses, so why would you have timesheets for unpaid work in the payroll app =)

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -213,12 +223,8 @@ structure (such as :guilabel:`Regular Pay`) to view all the rules.
-.. image:: payroll/new-rule.png
-   :align: center
-   :alt: Enter the information for the new rule.
+To make a new rule, click :guilabel:`New`. A new rule form appears. Enter the information in the
+fields.
```
I checked in both 16.4 and 16.3 but the info I have looks correct- you don't scroll down, you just click new at the top. So this does work.

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -91,32 +88,41 @@ form:
+.. image:: payroll/new-work-entry.png
+   :align: center
+   :alt: New work entry type form.
+
```
This is now included- I had it under 'check boxes' - this was how it was originally worded, so I just wrote in 'check the box if it counts' but now they've been fleshed out  bit more and the sections defined clearly.

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -164,9 +175,8 @@ There are two default structure types configured in Odoo: *Employee* and *Worker
-all fields can be edited. Once the fields are edited, click the :guilabel:`Save` button to save the
-changes, or click :guilabel:`Discard` to delete the entry.
+Click the :guilabel:`New` button to make a new structure type. Most fields are pre-populated, but
+all fields can be edited.
```
Added a whole new section! Nice catch.

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -185,13 +195,13 @@ useful to add may be a `Bonus`.
 
+Each :ref:`structure type <payroll/structure-types>` lists the various structures associated with
+it. Each structure contains a set of rules that define it.
+
```
I thought this section read well, and I believe the images could be removed and it would still make sense (that's how I test in my mind if it's explained enough or not). I did keep it simple, but I think concise.

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -238,6 +244,10 @@ The required fields for a rule are:
   Code`. Depending on what is selected, the fixed amount, percentage, or Python code needs to be
```
I am a bit unsure about the first part of this comment- I can't tell what you are asking me to change. As for the instructions, I thought they were brief but explained enough- I can expand on some if you call any specific ones out?

larm-odoo commented on content/applications/hr/payroll.rst:
Yes! I asked and that's possible. I added it with a link to the company setup doc,

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -130,8 +136,13 @@ by typing in the time.
    The :guilabel:`Work From` and :guilabel:`Work To` times must be in a 24-hour format. For example,
```
Doesn't do anything- just changes the order and does not affect how the info is tallied/quantified. I did not add this as I think it's fairly self explanatory and dragging or moving the order of the days doesn't do anything.

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -130,8 +136,13 @@ by typing in the time.
+.. image:: payroll/new-working-schedule.png
+   :align: center
+   :alt: New working schedule form.
 
```
I believe the employee has their company set and time zone set and just enters their time sheets as usual. So they know what working hours should be for them, as well as where they are located. I am going to put a pin in this since the other docs don't mention this in more detail. I asked Quentin about this. From what I believe, you'd enter your worked hours whatever they are for you (like 9-5) and Odoo should know where the employee is located on their card, and take the time for their work base don that. IF for some reason this is a complicated issue and needs a lot more explanation that what is here, I will come back and edit this, but since this was OK for the other versions and there was no issue form the product people on what I wrote, I will leave this alone for the moment.

larm-odoo commented:
Hi @tiku-odoo - I addressed all the comments you made- thanks for the thorough review. I replied to all the comments with what I did or the reasoning behind what I did. all set for a second look!

StraubCreative commented:
> Hi @tiku-odoo - I addressed all the comments you made- thanks for the thorough review. I replied to all the comments with what I did or the reasoning behind what I did. all set for a second look!

@larm-odoo re: the parts where Tim is talking about _meaningful annotations_:
Ideally, screenshots have fields/selections already filled in, in order to give readers a success model to follow. As well, highlight specific features with red boxes to capture attention on the UI as it relates to the documentation. Here are some examples:
- [highlighting features on the UI](https://www.odoo.com/documentation/16.0/applications/inventory_and_mrp/manufacturing/management/kit_shipping.html)
- [filled out forms](https://www.odoo.com/documentation/16.0/applications/inventory_and_mrp/manufacturing/quality_control/quality_control_points.html)

(the best screenshots feature both imo— red box highlights AND filled out form/fields)

larm-odoo commented:
Will do @StraubCreative - ok @tiku-odoo, please hold while I update the screens and I'll let you know when to come back!

larm-odoo commented:
Alright @tiku-odoo - images have been updated- I used the real set payroll items so the info was accurate. All set (for real this time!). =)

StraubCreative commented:
Don't forget to re-request reviews and to approve on to the next stage 😉

tiku-odoo reviewed (commented):
@larm-odoo Here is a partial review, I will complete the rest tomorrow. Please hold off on making any changes. 

I'd like to note that the doc looks a lot better. Great flow and explanation so far. I'm still missing the highlights in some of the images. 

Also to note, you mentioned that this doc was modeled after saas-16.4

You will need to change the branch you rebase to-- it's currently 16.0

Here is the process per JCS: 
1. put the PR in draft.
2. locally, you'll have to change the base of your branch on your computer (I'll explain later if you need it)
3. force push
4. on the pr, change the target to saas-16.3 then make the PR ready for review again
 
![draft](https://github.com/odoo/documentation/assets/91896312/f599786e-a7a7-41b6-bdf3-94428e7b11c4)

For Point 2: 
commit your changes
copy the commit hash number somewhere (you can find it by running git log)
git fetch origin
git reset --hard origin/saas-16.3
git cherry-pick [your commit's hash number]

and for point four, there is an "edit" button on your PR

![saas](https://github.com/odoo/documentation/assets/91896312/48c450eb-107f-49db-99c5-84a06ae65e3c)

then finally: 
git push --force-with-lease --set-upstream origin 16.0-payroll-targeted-updates-for-16-larm

![reset](https://github.com/odoo/documentation/assets/91896312/330ce33a-eb63-406b-a895-f287ed866b75)

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -238,29 +368,38 @@ The required fields for a rule are:
+:guilabel:`Code`, and which structure it applies to in the :guilabel:`Availability in Structure`
+field.
+
+The :guilabel:`Code` is used in the salary rules to compute payslips. If the :guilabel:`Availability
+in Structure` field is left blank, it indicates that the new input type is available for all
+payslips and is not exclusive to a specific structure.
```
shift sentence about availability in structure to above, or make as a important/note block

larm-odoo replied:
Made it an Important block.

tiku-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -286,21 +426,15 @@ When offering potential employees a position, there can be certain *advantages*
-To see the advantages, go to :menuselection:`Payroll --> Configuration --> Advantages`. Advantages
-are grouped by :guilabel:`Structure type`.
+To see the advantages, go to :menuselection:`Payroll --> Configuration --> Salary Package
+Configurator --> Advantages`. Advantages are grouped by :guilabel:`Structure type`, and the
+advantage listed for a particular structure type is only available for that specific structure.
```
This last sentence needs explanation it sounds confusing. Maybe an example block

larm-odoo replied:
Added an example! Might be verbose, but it makes this clear.

tiku-odoo reviewed (changes requested):
@larm-odoo I've finished my 2nd review of this doc. It's ready to move on after you make the changes I requested. Excellent work on the updates; they really make the doc flow much better now.
Looking forward to seeing this in our production. 

Once you push up the changes, ping me and I will approve to move on :+1: 

Tim

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -91,31 +109,69 @@ form:
 
-.. image:: payroll/working-times.png
+- :guilabel:`Time Off`: Check this box if the work entry type can be selected for a time off request
+  or entry in the *Time Off* application. If :guilabel:`Time Off` is checked off, a
```
That's a "me nomenclature" - I say 'checked off' which is confusing. Checked is better- I like it!

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -154,23 +215,96 @@ end of year bonus. Both the `Regular Pay` structure and `End of Year Bonus` stru
+  for salaried employees who receive the same wage every pay period, select :guilabel:`Monthly Fixed
+  Wage`. If the wage type is going to be used for employees who receive wages based on how many
+  hours they worked during a pay period, select :guilabel:`Hourly Wage`.
+  :guilabel:`Monthly Fixed Wage`.
```
Yup- it was a placeholder for all the fields in the list but I didn't notice it wasn't configured from the RST when I looked through. Good catch!

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -154,23 +215,96 @@ end of year bonus. Both the `Regular Pay` structure and `End of Year Bonus` stru
+  will fall under from the drop-down menu. Options include :guilabel:`Attendance`,
+  :guilabel:`Overtime Hours`, :guilabel:`Generic Time Off`, :guilabel:`Compensatory Time Off`,
+  :guilabel:`Home Working`, :guilabel:`Unpaid`, :guilabel:`Sick Time Off`, :guilabel:`Paid Time
+  Off`, and :guilabel:`Out Of Contract`.
```
Hi Tim - I double checked and this is correct. There are 7defauly options for "Default Scheduled Pay"- both in runbot and in a blank test database I created to double check.

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -154,23 +215,96 @@ end of year bonus. Both the `Regular Pay` structure and `End of Year Bonus` stru
+Working Hours` field on the new structure type form. Click :guilabel:`Create and edit`. A default
+working hours form will pop up. The default working hours form has two sections, a general
+information section and a tab listing out all the individual working hours by day and time. When the
+form is filled out, click :guilabel:`Save & Close`.
```
I double checked- this does appear on this form. I know most of the time you don't have to save, but this does appear. 
![image](https://github.com/odoo/documentation/assets/121518652/1a3da376-c95a-4725-8103-e1cf783dcdf2)

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -154,23 +215,96 @@ end of year bonus. Both the `Regular Pay` structure and `End of Year Bonus` stru
+- :guilabel:`Company Full Time`: Enter the number of hours per week an employee would need to work
+  in order to be considered a full-tme employee. Typically, this is approximately 40 hours, and this
+  number affects what types of benefits an employee can receive based on their employment status
+  (full time vs party time).
```
Hahaha- my spell check nemesis- when the word is wrong but not flagged because it's a real word!

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -154,23 +215,96 @@ end of year bonus. Both the `Regular Pay` structure and `End of Year Bonus` stru
+     working hours form. This will change the working hours tab to display two weeks of working
+     times that can be adjusted.
 
 Structures
```
Just structures- that's why the introduction explains the difference between the two/how they relate .

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -311,49 +445,86 @@ The required fields for an advantage are:
+The required fields, aside from entering the :guilabel:`Information` name, are
+:guilabel:`Related Field` and :guilabel:`Category`. Select a :guilabel:`Related Field` from the
+drop-down menu that best describes what kind of personal information this entry is, and where it is
+going to be stored in the backed. Select a :guilabel:`Category` from the drop-down menu that the
```
Thanks Tim- I had Sam install one for me. I still sometimes miss the "not incorrect but wrong" words, like "party" instead of "part", when the doc is this long and my eyes glaze over. I will be more diligent about that!

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -206,19 +340,15 @@ Each structure has a set of *salary rules* to follow for accounting purposes. Th
+Click on a structure (such as :guilabel:`Regular Pay`) to view all the rules.
 
 .. image:: payroll/rules.png
    :align: center
-   :alt: Rules for each salary structure type.
+   :alt: All the rules for each salary structure type, organized by salary structure type.
```
True- it is very similar, only a slight difference. I'll remove it, it is redundant- especially because the image is so LONG.

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -206,19 +340,15 @@ Each structure has a set of *salary rules* to follow for accounting purposes. Th
-   :alt: Enter the information for the new rule.
+To make a new rule, click :guilabel:`New`. A new rule form appears. Enter the information in the
+fields.
 
```
Added some more info as to why (reporting and payroll processing)..

larm-odoo commented on content/applications/hr/payroll.rst:
```diff
@@ -311,49 +445,86 @@ The required fields for an advantage are:
-.. image:: payroll/personal-new.png
-   :align: center
-   :alt: New personal information entry.
+The required fields, aside from entering the :guilabel:`Information` name, are
+:guilabel:`Related Field` and :guilabel:`Category`. Select a :guilabel:`Related Field` from the
```
Wow - I checked this several times, in runbots in 16-16.4, and for whatever reason I missed this! Also, I can't NOT select one, so that is also why I missed this. I try to delete everything, click save, to see what's required- but thisfield is not possible to leave blank. Very good catch- thank you!

larm-odoo commented:
Hi @tiku-odoo - changes implemented and ready for your approval. Thanks for all your reviews!

tiku-odoo commented:
@larm-odoo  Thanks for making those changes. I've approved, the copy you've made. However- I think you should talk to @StraubCreative or @samueljlieber about the branch your commit into (16.0) is not the correct version that I reviewed this document for. saas-16.4 should be the version you're pushing into. 

See above comment on how to change this in your PR: https://github.com/odoo/documentation/pull/5449#pullrequestreview-1636464596

larm-odoo commented:
Hey @tiku-odoo and @StraubCreative - just an update for this. After speaking with Sam, since this doc is fully reviewed and edited (from JEOR and TIKU) and ready for 16.4, this doc will be pushed for 16.0 all the way up to master. Then, I will make a PR to make any modifications needed for 16.0 since that will need a few slight changes. I think this would be easier than me changing this to 16.0, then doing another PR to add this current version for 16.4. So that's the plan! =)

samueljlieber reviewed (approved):
Hi @larm-odoo great job on the improvements in this PR. I am approving now 👍

@StraubCreative this PR is good to go!

StraubCreative reviewed (approved):
@robodoo r+
